Security/src/EOMT.ps1

```diff
@@ -600,6 +603,31 @@ function Get-ServerVulnStatus {
     return $false
 }
 
+function Get-PatchMessage {
```

Comment:
This should also be called in Write-Summary

Reply:
I didn't say to rename this to Write-Summary, I said to call this from within the already-existing function Write-Summary